src/04_printing.py

```diff
@@ -11,7 +11,10 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("x is %s, y is %s, z is \"%s\"" %(x, round(y,2), z))
 
 # Use the 'format' string method to print the same thing
+print("x is {}, y is {}, z is \"{}\"".format(x, round(y,2), z))
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+# Finally, print the same thing using an f-string
+print(f"x is {x}, y is {round(y,2)}, z is \"{z}\"")
```

Comment:
Love the escape chars `\` here. This can also be accomplished by using a combination of `"` and `'` chars but it is a matter of preference.